euth/comments/autofixtures.py

```diff
@@ -0,0 +1,19 @@
+from django.contrib.contenttypes.models import ContentType
+
+from .models import Comment
+from home.models import HomePage
+from autofixture import generators, register, AutoFixture
+
+class CommentAutoFixture(AutoFixture):
+
+    homepage_contenttype = ContentType.objects.get(app_label='home', model='homepage')
+    homepage_id = HomePage.objects.first().pk
+
+    field_values = {
+        'content_type': homepage_contenttype,
+        'object_pk': homepage_id,
```

Comment:
@keimlink Is there a good way to build a auto-fixture for generic forgein keys? I imagine a function that takes a list of models and will automatically select a random instance from those models and append a comment to it. 

The function seems trivial but how do I set fixture values that are somehow connected?